cocos/core/components/component-event-handler.ts

```diff
@@ -30,7 +30,7 @@
  */
 
 import { ccclass, type, serializable, editable, tooltip } from 'cc.decorator';
-import type { Node } from '../scene-graph';
+import { Node } from '../scene-graph';
```

Comment:
Can not modify it, or it will meet error in native.
@dumganhar i remembered you modify it for native.

Reply:
Yes, I remember that @shrinktofit suggested to add type keyword here. It seems to resolve cycle reference issues.

What errors did you get if the type keyword is not removed?

Reply:
Error is  `@type(Node)`  would not work

Reply:
If there is not cycle reference issue. Then it is not problem.